### PR TITLE
[Fix]: Bar Graph tooltip issue

### PR DESCRIPTION
**Pull request by nxanil**
@killanch @preshitg

Fixed issue #423. Please have a look.

barGraphTooltip

### CODE REVIEW

**Comment by preshitg** [comment]
@nxanil Thank you. I will verify this and let you know.

File: Graphs/BarGraph/index.js [hunk 1]
```diff
@@ -117,6 +119,9 @@ const BarGraph = (props) => {
         }
     );
 
+    if (!parsedData || !parsedData.length) {
+        return renderMessage({ message: "No data to visualize", id, classes });
```
Comment (preshitg) [inline Graphs/BarGraph/index.js:123]: Is this change required for this bug?
In any case, can you use `NO_DATA_FOUND` from constants?

**Comment by preshitg** [comment]
> @nxanil Thank you. I will verify this and let you know.

Checked BarGraph tooltip under Domain/VSS Analytics/Service Analytics/Top 5 services and it looks good. 
Please address one minor comment and then we can merge this.

File: Graphs/BarGraph/index.js (hunk 1 above)
Comment (nxanil) [reply Graphs/BarGraph/index.js]: Done

**Review by killanch — APPROVED** [review]

**Comment by killanch** [comment]
@preshitg If ok, pls merge. Pls bring it to r20.10 if same issue exists. Pls also raise a JIRA ticket for tracking.

**Review by preshitg — APPROVED** [review]
